source/javascripts/directives/multi_select.js

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
Ok :)

It's just strange to have some files as js and some as coffeescript in the app. But I guess translating the directive to coffeescript would be too much work.